utils/json-utils/src/main/scala/pl/touk/nussknacker/engine/json/swagger/SwaggerTyped.scala

```diff
@@ -39,7 +39,7 @@ case class SwaggerEnum(values: List[String]) extends SwaggerTyped
 
 case class SwaggerArray(elementType: SwaggerTyped) extends SwaggerTyped
 
-case class SwaggerObject(elementType: Map[PropertyName, SwaggerTyped]) extends SwaggerTyped
+case class SwaggerObject(elementType: Map[PropertyName, SwaggerTyped], additionalPropertiesOn: Boolean, additionalPropertiesType: Option[SwaggerTyped]) extends SwaggerTyped
```

Comment:
can we model this like that:
```
case class SwaggerObject(elementType: Map[PropertyName, SwaggerTyped], additionalProperties: Either[Boolean,SwaggerTyped]) extends SwaggerTyped
```

because now you can set `additionalPropertiesOn = false` and ant the same time `additionalPropertiesType = Some(String)` which is confusing

Reply:
Changed